waku/v2/node/wakunode2.nim

```diff
@@ -87,6 +88,7 @@ type
     filters*: Filters
     rng*: ref BrHmacDrbgContext
     wakuDiscv5*: WakuDiscoveryV5
+    announcedAddress* : seq[MultiAddress]
```

Comment:
```suggestion
    announcedAddresses* : seq[MultiAddress]
```
Plural variable name for sequences. If you prefer using abbreviations, we've settled on `addr` for a singular address and `addrs` for more than one. Just make sure it is consistent with other uses of `announcedAddresses`.

Reply:
fixed in 5eff5597eabec4afaca39814bcd04d751c41ed2f